frameworks/lightautoml/exec.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import os
+
+import pandas as pd
+import numpy as np
+import pickle
+import warnings
+
+from frameworks.shared.callee import call_run, result, output_subdir, utils, save_metadata
+from lightautoml.tasks import Task
+from lightautoml.automl.presets.tabular_presets import TabularAutoML, TabularUtilizedAutoML
+from lightautoml import __version__
```

Comment:
```suggestion
import logging
import os
import pickle
import warnings

import matplotlib
import numpy as np
import pandas as pd
matplotlib.use("agg")  # no need for tk

from lightautoml.tasks import Task
from lightautoml.automl.presets.tabular_presets import TabularAutoML, TabularUtilizedAutoML
from lightautoml import __version__

from frameworks.shared.callee import call_run, result, output_subdir, utils, save_metadata
```
I'm being picky here,  but just grouping import by usual conventions (standard libs to most specific libraries) and using lexicographic order.

Also wanted to be able to run it directly on macOS: as you're internally using matplotlib, using `agg` backend is usually enough to make code portable on macOS.
However, it looks like you're spawning subprocesses and those fail with the usual "No module named _tkinter" on macOS, so the backend would have to be set on your side.
Not making it a requirement, but I think your users would be happy to be able to run your tool directly on macOS.

Reply:
> Also wanted to be able to run it directly on macOS
Ok, I will fix that in our library, thank you!